### DISCUSSION TITLE
Move multiple charts at the same time

### DISCUSSION THREAD

**Discussion by ondolist** [reactivemarkets/react-financial-charts#625] [Q&A]
I am trying to make a program that shows several charts at the same time. All charts should show the same interval of data.
Is there a way to move multiple charts at the same time? (Zoom in, zoom out, and drag should be applied to all charts at the same time.)

Or, is there a way to find out which data currently spans leftmost and rightmost when the chart is dragged?

**Comment by Arnau-Gelonch** [comment]
Has anyone been able to solve it?

**Comment by paulbse** [comment]
@ondolist did you find a way to do it ?

**Reply by paulbse** [reply]
i think i found a way to do it

**Reply by dnjsgur0629** [reply]
can you share the way you found?
